src/js/utils/skin.js

```diff
@@ -0,0 +1,219 @@
+
+        return config;
+    }
+
+    return colors;
+}
+
+export function handleColorOverrides(playerId, skin = {}) {
+
+    function addStyle(elements, attr, value, extendParent) {
+        if (!value) {
+            return;
+        }
+        /* if extendParent is true, bundle the first selector of
+         element string to the player element instead of defining it as a
+         child of the player element (default). i.e. #player.sel-1 .sel-2 vs. #player .sel-1 .sel-2 */
+        elements = prefix(elements, '#' + playerId + (extendParent ? '' : ' '));
+
+        const o = {};
+        o[attr] = value;
+        css(elements.join(', '), o, playerId);
+    }
+
+    // These will use standard style names for CSS since they are added directly to a style sheet
+    // Using background instead of background-color so we don't have to clear gradients with background-image
+
+    if (_.size(skin.controlbar)) {
+        styleControlbar(skin.controlbar);
+    }
+    if (_.size(skin.timeslider)) {
+        styleTimeslider(skin.timeslider);
+    }
+    if (_.size(skin.menus)) {
+        styleMenus(skin.menus);
+    }
+    if (_.size(skin.tooltips)) {
+        styleTooltips(skin.tooltips);
+    }
+
+    insertGlobalColorClasses(skin.menus);
+
+    function styleControlbar(config) {
+
+        addStyle([
+            // controlbar button colors
+            '.jw-controlbar .jw-text',
+        ], 'color', config.text);
+
+        addStyle([
+            // controlbar button colors
+            '.jw-button-color:not(.jw-icon-cast)',
+            '.jw-button-color.jw-toggle.jw-off:not(.jw-icon-cast)',
+        ], 'color', config.icons);
+
+        // Apply active color
+        addStyle([
+            // Toggle and menu button active colors
+            '.jw-button-color.jw-toggle:not(.jw-icon-cast)',
+            '.jw-button-color:hover:not(.jw-icon-cast)',
+            '.jw-button-color.jw-toggle.jw-off:hover:not(.jw-icon-cast)'
+        ], 'color', config.iconsActive);
+
+        // Chromecast overrides
+        // Can't use addStyle since it will camel case the style name
+        css('.jw-icon-cast button.jw-off', `{--disconnected-color: ${config.icons}}`, playerId);
+        css('.jw-icon-cast:hover button.jw-off', `{--disconnected-color: ${config.iconsActive}}`, playerId);
+        css('.jw-icon-cast button.jw-off:focus', `{--disconnected-color: ${config.iconsActive}}`, playerId);
+
+        css('.jw-icon-cast button', `{--connected-color: ${config.iconsActive}}`, playerId);
+        css('.jw-icon-cast button:focus', `{--connected-color: ${config.iconsActive}}`, playerId);
+        css('.jw-icon-cast:hover button', `{--connected-color: ${config.iconsActive}}`, playerId);
+
+        addStyle([
+            '.jw-background-color.jw-controlbar'
+        ], 'background', config.background);
+    }
+
+    function styleTimeslider(config) {
+
+        addStyle([
+            '.jw-progress',
+            '.jw-buffer',
```

Comment:
This was a mistake in the mock, itll the same color, just a different transparency